zokrates_core/build.rs

```diff
@@ -25,11 +25,17 @@ fn main() {
             Repository::clone(LIBSNARK_URL, libsnark_source_path).unwrap()
         });
 
-        let commit = Oid::from_str(LIBSNARK_COMMIT).unwrap();
-        let commit = repo.find_commit(commit).unwrap();
+        // Unencrypted `git://` protocol is no longer supported on GitHub
+        // so we replace all submodule urls to use `https://`
+        let gitmodules_path = libsnark_source_path.join(".gitmodules");
+        let gitmodules = std::fs::read_to_string(&gitmodules_path)
+            .unwrap()
+            .replace("git://", "https://");
```

Comment:
This code can be removed once the following PR is merged: https://github.com/scipr-lab/libsnark/pull/192